resource/reactivedrop_swedish.txt

```diff
@@ -2223,7 +2223,7 @@
 "asw_instructor_heal_self"		"Hela Dig själv"
 "[english]asw_instructor_request_heal"		"Call Medic"
 "asw_instructor_request_heal"		"Kalla på Sjukvårdare"
-"[english]asw_instructor_clip_usage"		"%s1 Ammo lost in old clip"
+"[english]asw_instructor_clip_usage"		"%s1 Ammo lost in old mag"
 "asw_instructor_clip_usage"		"%s1 Ammo förlorad i det gamla magasinet"
```

Comment:
```suggestion
"asw_instructor_clip_usage"		"%s1 ammunition förlorad i det gamla magasinet"
```
Capital ammo is weird when it says for example "50 Ammo förlorad i det gamla magasinet", will you just quickly change it for me?